pkg/archive/archive.go

```diff
@@ -426,8 +426,14 @@ func createTarFile(path, extractDir string, hdr *tar.Header, reader io.Reader, L
 	}
 
 	for key, value := range hdr.Xattrs {
+		hadError := false
 		if err := system.Lsetxattr(path, key, []byte(value), 0); err != nil {
-			return err
+			// We ignore errors here because not all graphdrivers support xattrs.
```

Comment:
I think aggregating into one error is more consistent with other places where we do that.